app/models/user.js

```diff
@@ -55,5 +63,12 @@ module.exports = (sequelize, DataTypes) => {
       throw errors.databaseError(err);
     });
 
+  User.createAdmin = user => {
+    return User.update({ roleUser: user.roleUser }, { where: { email: user.email } }).catch(err => {
```

Comment:
Remove brackets and return, also this only update the role what happen if name is changed too ?